script/Ciel/Day 3/03_01_CIEL03_3_3B.txt

```diff
@@ -292,92 +305,102 @@ Ah...
 [sha:64cf22d362e618d79bf570d226397b9ac1367b04]{
 -- Page 210, Offset 5396.
 -- また先輩にはみっともないところを見られたけど、思考も視界も、今はいたって正常だ。
-Senpai saw me in a disgraceful state again, but my thoughts and vision are back to normal now.
+I've let Senpai see me at my worst again,
+ but at least now my thoughts and vision are back to normal.
 }
 [sha:8511aeda81f2de6cedbb97037479a6b3fe973643]{
 -- Page 211, Offset 5397.
 -- 「良かった。落ち着きましたか、遠野くん」
-"Good. Have you calmed down, Tohno-kun?"
+"Thank goodness. Have you calmed down, Tohno-kun?"
 }
 [sha:21140fc7359bb6e1f21827d1634bceb6f80ace42]{
 -- Page 211, Offset 5398.
 -- 顔をあげると、そこにはいつもの先輩の姿がある。
-I look up and see Senpai standing there, the way she always does.
+// Reading いつもの先輩 as no-lines Senpai
+Glancing up I see Senpai standing there,
+ looking just as she always has.
 }
 [sha:0d73c4abc36b2c55563e5bf122fc7d0e85f1e783]{
 -- Page 212, Offset 5399.
 -- 「ほら、やっぱり満足に歩けないじゃないですか。
-"There, you still can't walk properly right?
+"See, you still can't walk properly after all, can you?
 }
 [sha:d5479c8444b15796b831cea5381a7f4a6292a27e]{
 -- Page 212, Offset 5400.
 -- 体が冷えきっているコト、分かってくれました？」
-Your body got really cold, you understand that, right?"
+Do you appreciate just how frozen through your body is now?"
```

Comment:
I like "appreciate" but "how frozen through your body is" is an unusual expression. This is a very simple expression so
> Do you appreciate now just how cold your body [got/is]?